src/site/stages/build/process-cms-exports/schemas/transformed/node-health_care_region_page.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-health_care_region_page'] },
+    entity: {
+      type: 'object',
+      properties: {
+        entityType: { enum: ['node'] },
+        entityBundle: { enum: ['health_care_region_page'] },
+        title: { type: 'string' },
+        entityUrl: {
+          type: 'object',
+          properties: {
+            // TODO: add breadcrumb
+            path: { type: 'string' },
+          },
+          required: ['path'],
+        },
+        fieldNicknameForThisFacility: { type: 'string' },
+        // TODO: Figure out the type vs. __typename stuff
+        entityMetaTags: {
+          type: 'array',
+          items: {
+            type: 'object',
+            properties: {
+              // __typename: { type: 'string' },
```

Comment:
Should this still be commented? If we're using `__typename` right now, we should declare it.

Also might be worth adding an `additionalProperties: false` on here to make sure we catch any future refactoring. (Specifically, if we change `__typename` to `type` down the road, we want to make sure the schema still covers it.)